### ISSUE TITLE
Cannot install pydot for Python3 via pip

### ISSUE THREAD

**Issue by nvdv** [erocarrera/pydot#110]
```pip3 install pydot``` returns 
```
Collecting pydot
  Downloading pydot-1.0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/var/folders/35/xqrmy3y91wz2zzs0c33jl4m00000gn/T/pip-build-vtolqv73/pydot/setup.py", line 5
        except ImportError, excp:
                          ^
    SyntaxError: invalid syntax

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/35/xqrmy3y91wz2zzs0c33jl4m00000gn/T/pip-build-vtolqv73/pydot
```

**Comment by JWarmenhoven** [comment]
That's because the version on PyPi is not (yet) compatible with Python3.
I see there is a pull request (#106) pending for Python 3.5 support.

Check out https://github.com/nlhepler/pydot for a Python3 compatible version.

**Comment by log0** [comment]
TL;DR: pip3 install pydot3 / pip install pydot3

I found that this repro doesn't work in Python 3.x and made a quick fix to be Python 3.x compatible and made it accessible via pip install.

**Comment by johnyf** [comment]
Should be fixed by #76 and #96.

**Comment by nvdv** [comment]
Great!

**Comment by freelighting** [comment]
in my mac, it can be installed with pip(but it is not work with conda install)
